backend/pyproject.toml

```diff
@@ -23,6 +23,7 @@ omit = [
 [tool.poetry.dependencies]
 algoliasearch = "^4.13.0"
 algoliasearch-django = "^4.0.0"
+beautifulsoup4 = "^4.12.2"
```

Comment:
As we already have lxml installed can we handle it with xpath instead of adding a new dependency?